includes/reader-revenue/woocommerce/my-account/class-woocommerce-my-account.php

```diff
@@ -79,9 +72,28 @@ public static function init() {
 			\add_filter( 'wc_memberships_my_memberships_column_names', [ __CLASS__, 'remove_next_bill_on' ], 21 );
 			\add_action( 'profile_update', [ __CLASS__, 'handle_admin_email_change_request' ], 10, 3 );
 			\add_action( self::SYNC_ESP_EMAIL_CHANGE_CRON_HOOK, [ __CLASS__, 'sync_email_change_with_esp' ], 10, 3 );
+
+			// Decide which My Account UI version to load.
+			if ( version_compare( self::get_version(), '1.0.0', '<' ) ) {
+				include_once NEWSPACK_ABSPATH . 'includes/reader-revenue/woocommerce/my-account/class-my-account-ui-v0.php';
+			} else {
+				include_once NEWSPACK_ABSPATH . 'includes/reader-revenue/woocommerce/my-account/class-my-account-ui-v1.php';
+			}
```

Comment:
Do we need the full path here? I think this should be fine:

```php
include_once __DIR__ . '/class-my-account-ui-v0.php';
```

Reply:
Updated in ae456e80